src/Meshes.jl

```diff
@@ -78,6 +79,8 @@ export
   # connectivities
   Connectivity,
   polytopetype, connect, materialize,
+  OrderingConvention, GMSH,
+  connectivity,
```

Comment:
I don't think the ordering convention should be placed in the same section of the connectivities. It deserves a separate section with conventions, which is a concept in itself.